storage_service/locations/api/resources.py

```diff
@@ -381,19 +381,19 @@ def post_detail(self, request, *args, **kwargs):
                     origin_location.relative_path, source_path)
                 destination_path = os.path.join(
                     destination_location.relative_path, destination_path)
-                origin_space.move_to_storage_service(
+                origin_space.move_direct(
```

Comment:
Should we be trying `move_direct` and falling back to the old way if a `NotImplementedError` is raised?

Reply:
Thanks @jrwdunham, appreciate the review.

Great idea, thanks.